frontend/src/components/layout/NavBar.js

```diff
@@ -1,32 +1,53 @@
-import React from "react";
+import React, { useContext } from "react";
 import { NavLink } from 'react-router-dom';
+import { AxiosInstance } from "../../utils/Helpers";
 import styled from "styled-components";
-
+import AuthContext from "../../context/AuthContext";
 import CheckInput from '../elements/CheckInput';
 
 const StyledNav = styled.nav`
   margin-top: 1rem;
+  padding: 1rem 3rem;
   width: 100%;
   height: 100%;
   display: flex;
-  > a {
+  justify-content: space-between;
+  align-items: center;
+  div > a {
     color: ${({theme}) => theme.onPrimary};
     font-size: 1.25rem;
     justify-self: center;
   }
-  > a ~ a {
+  div > a ~ a {
     margin-left: 1rem;
   }
 `;
 
-const NavBar = ({ onChange, theme, className, children }) => {
-  
+const NavBar = ({ onChange, theme, className, setError, setErrorMessage, setSuccess, setSuccessMessage }) => {
+  const { loggedIn, setLoggedIn, setUser } = useContext(AuthContext)
+
+  const logUserOut = async() => {
+    const { status } = await AxiosInstance.post('users/logout')
+    if(status === 200){
+        setLoggedIn(false)
+        setSuccess(true)
+        setUser({})
+        setSuccessMessage('See ya!')
+    }
+  }
+
   return (
     <StyledNav className={className}>
+      <div>
         <NavLink to="/">Home</NavLink>
-        <CheckInput theme={theme} onChange={onChange} />
+        <CheckInput checked={null} theme={theme} onChange={onChange} />
```

Comment:
Why is this `checked` property ALWAYS null? If it's a `false` value, you can also remove it

Reply:
how do I deal with the controlled uncontrolled thing?